__docs__/wonder-blocks-pill/pill.stories.tsx

```diff
@@ -334,52 +322,28 @@ Variants.play = async ({canvasElement}) => {
     });
 
     // Test clickable pill styles
-    await userEvent.tab();
-    let computedStyle = getComputedStyle(neutralSmallClickable, ":hover");
-    await expect(computedStyle.outline).toBe("rgb(24, 101, 242) solid 2px");
-
-    await userEvent.tab();
-    computedStyle = getComputedStyle(accentSmallClickable, ":hover");
-    await expect(computedStyle.outline).toBe("rgb(24, 101, 242) solid 2px");
-
-    await userEvent.tab();
-    computedStyle = getComputedStyle(infoSmallClickable, ":hover");
-    await expect(computedStyle.outline).toBe("rgb(24, 101, 242) solid 2px");
-
-    await userEvent.tab();
-    computedStyle = getComputedStyle(successSmallClickable, ":hover");
-    await expect(computedStyle.outline).toBe("rgb(24, 101, 242) solid 2px");
-
-    await userEvent.tab();
-    computedStyle = getComputedStyle(warningSmallClickable, ":hover");
-    await expect(computedStyle.outline).toBe("rgb(24, 101, 242) solid 2px");
-
-    await userEvent.tab();
-    computedStyle = getComputedStyle(criticalSmallClickable, ":hover");
-    await expect(computedStyle.outline).toBe("rgb(217, 41, 22) solid 2px");
-
-    await userEvent.tab();
-    computedStyle = getComputedStyle(neutralLargeClickable, ":hover");
```

Comment:
Removed these because I realized the tests were redundant. It was testing the same style block for all the different sizes.